File: XPile 3-17/Test/Integration/mathlib.js
function factorial(n) {
    if ((n <= 1)) {
        return 1;
    }
    return (n * factorial((n - 1)));
}

function square(x) {
    return (x * x);
}

function add(a, b) {
    return (a + b);
}


main();

File: XPile 3-17/Test/Integration/mathlib.py
from dataclasses import dataclass, field


def factorial(n):
    if (n <= 1):
        return 1
    return (n * factorial((n - 1)))

def square(x):
    return (x * x)

def add(a, b):
    return (a + b)


if __name__ == "__main__":
    main()
